packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,29 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
 
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-center() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
hmm my primary hesitation with these mixins is that they have nothing to do with table loading. they are a) more generally useful but b) not complex or interesting. kinda suggests they're unnecessary overhead and we'd be better off without them in this PR.